qiskit_optimization/translators/docplex_mp.py

```diff
@@ -364,22 +419,4 @@ def from_docplex_mp(model: Model, indicator_big_m: Optional[float] = None) -> Qu
         elif not isinstance(constraint, (QuadraticConstraint, IndicatorConstraint)):
             raise QiskitOptimizationError(f"Unsupported constraint: {constraint}")
 
-    # get linear constraints
-    for constraint in model.iter_linear_constraints():
-        lhs, sense, rhs = _FromDocplexMp._linear_constraint(var_names, constraint)
-        quadratic_program.linear_constraint(lhs, sense, rhs, constraint.name)
-
-    # get quadratic constraints
-    for constraint in model.iter_quadratic_constraints():
-        linear, quadratic, sense, rhs = _FromDocplexMp._quadratic_constraint(var_names, constraint)
-        quadratic_program.quadratic_constraint(linear, quadratic, sense, rhs, constraint.name)
-
-    # get indicator constraints
-    for constraint in model.iter_indicator_constraints():
-        linear_constraints = _FromDocplexMp._indicator_constraints(
-            var_names, var_bounds, constraint, indicator_big_m
-        )
-        for linear, sense, rhs, name in linear_constraints:
-            quadratic_program.linear_constraint(linear, sense, rhs, name)
-
-    return quadratic_program
+    return _FromDocplexMp(model).quadratic_program(indicator_big_m)
```

Comment:
Do we need to add info about the added support for logical expressions into the docstring of this `from_docplex_mp` method

Reply:
Thank you. I update the docstring.